build/android/BUILD.gn

```diff
@@ -30,6 +30,19 @@ action("sign_app") {
   ]
 }
 
+action("collect_objs_app") {
```

Comment:
I think this should be part of `create_dist` for android

Reply:
it was moved there